src/app/researcher/study/request/[memberSlug]/study-proposal.tsx

```diff
@@ -5,19 +5,126 @@ import { Button, Group, Stack, Text } from '@mantine/core'
 import { notifications } from '@mantine/notifications'
 import { CancelButton } from '@/components/cancel-button'
 import { useForm } from '@mantine/form'
-import { StudyProposalFormValues, studyProposalSchema, zodResolver } from './study-proposal-schema'
+import { StudyProposalFormValues, studyProposalSchema } from './study-proposal-schema'
 import { StudyProposalForm } from './study-proposal-form'
 import { UploadStudyJobCode } from './upload-study-job-code'
 import { useMutation } from '@tanstack/react-query'
 import { onCreateStudyAction } from './actions'
 import { useRouter } from 'next/navigation'
+import { StudyDocumentType } from '@/lib/types'
+import { zodResolver } from 'mantine-form-zod-resolver'
+import { CodeReviewManifest } from '@/lib/code-manifest'
+import { PresignedPost } from '@aws-sdk/s3-presigned-post'
+import { signedUrlForCodeUploadAction, signedUrlForStudyFileUploadAction } from '@/server/actions/s3.actions'
 
-export const StudyProposal: React.FC<{ memberId: string }> = ({ memberId }) => {
+// TODO @nathan, should we talk about local vs s3 storage?
+//  Could we just use s3 locally and not have to switch on USING_S3 to simplify?
```

Comment:
discussion not here (commenting for visibility)

Reply:
I added to the dev agenda to discuss, I'm hesitant to require it because:
* I want to be able to develop without or with slow internet
* I don't think we want to have a requirement that all devs must have a AWS account